patches/minecraft/net/minecraft/entity/player/ServerPlayerEntity.java.patch

```diff
@@ -38,10 +39,11 @@
 -         this.field_70128_L = false;
 +         serverworld.removeEntity(this, true); //Forge: the player entity is moved to the new world, NOT cloned. So keep the data alive with no matching invalidate call.
 +         this.revive();
++         Entity e = teleporter.placeEntity(this, serverworld, serverworld1, this.field_70177_z, spawnPortal -> {
```

Comment:
This entity seems to be unused, it should be returned at the end of the function

Reply:
I was thinking about checking if the retuned entity is the same as this (so the player) because the player should not ever be recreated.

Reply:
Have you decided what to do with this? It seems that the basic value would just be to return e instead of this below.

Reply:
I added a check and throw an excpetion if the retuned enity is not the same player.